frontend/src/admin/__tests__/user-list.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import {fireEvent, render} from '@testing-library/react'
+import UserList from '../components/user-list/user-list'
+import { User } from '../models/user'
+import { describe, it } from '@jest/globals'
+import '@testing-library/jest-dom'
+
+describe('UserListItem', () => {
+  const users: User[] = [
+    {
+      id: 'test-johnDoe',
+      name: 'John Doe',
+      emberTarget: '/john-doe',
+      accessRightGroups: [{
+        name: 'sofie',
+        accessRights: [{
+          machineId: 'test',
+          label: 'test',
+          path: '/test',
+          anonymous: true
+        }]
+      }]
+    },
+    {
+      id: 'test-janeDoe',
+      name: 'Jane Doe',
+      emberTarget: '/jaen-doe',
+      accessRightGroups: [{
+        name: 'Qbox',
+        accessRights: [{
+          machineId: 'test',
+          label: 'test',
+          path: '/test',
+          anonymous: true
+        }]
+      }]
+    }
+  ]
+
+  it('renders user list correctly', () => {
+    const {getByText} = render(<UserList users={users} onClick={() => {console.log('Mock click')}}/>)
+    expect(getByText('Users')).toBeInTheDocument()
+    expect(getByText(users[0].name)).toBeInTheDocument()
+    expect(getByText(users[1].name)).toBeInTheDocument()
+  })
+
+  it('clicking "Download user" button calls downloadUser function', () => {
+    const downloadUserMock = window.alert = jest.fn()
+    const { getByText } = render(<UserList users={users} onClick={downloadUserMock} />)
+    fireEvent.click(getByText('Download user'))
+    expect(downloadUserMock).toHaveBeenCalled()
```

Comment:
It is good practice to be specific about what you expect. In this case: How many times to you expect the `downloadUserMock` to have been called? In this case it could potentially have been called a million times and the test will still pass.